### ISSUE TITLE
Mixup EPUB accessibility metadata techniques accessibilityFeature/accessMode

### ISSUE THREAD

**Issue by riekevullings** [w3c/publ-a11y#410]
There seems to be a mix-up of properties in the Display Techniques for EPUB Accessibility Metadata 2.0. The part Understanding the variables of 4.6 Charts, diagrams, and formulas refers to:

- accessibilityFeature="chartOnVisual" 
- accessibilityFeature="chemOnVisual" 

it should be:

- accessMode="chartOnVisual" 
- accessMode="chemOnVisual"

**Comment by clapierre** [comment]
This is fixed
